README.md

```diff
@@ -0,0 +1,96 @@
+# Indico [![Travis Build Status](https://travis-ci.org/indico/indico.svg?branch=master)](https://travis-ci.org/indico/indico) [![License](https://img.shields.io/github/license/indico/indico.svg)](https://github.com/indico/indico/blob/master/LICENSE) [![Available on PyPI](https://img.shields.io/pypi/v/indico.svg)](https://pypi.python.org/pypi/indico/) [![Made at CERN!](https://img.shields.io/badge/CERN-Open%20Source-%232980b9.svg)](https://home.cern)
+
+<img src="https://github.com/indico/indico/raw/master/indico/web/static/images/logo_indico.png"
+     align="right"
+     width="300"
+     style="width: 300px; float: right; margin-right: 50px;">
+
+**Indico** is:
+ * 🗓 a general-purpose **event management** tool;
+ * 🌍 fully **web-based**;
+ * 🧩 **feature-rich** but also **extensible** through the use of [plugins](https://docs.getindico.io/en/stable/plugins/);
+ * ⚖️ **Open-Source** Software under the MIT License;
+ * <img src="https://design-guidelines.web.cern.ch/sites/design-guidelines.web.cern.ch/files/u6/CERN-logo.jpg" width="20"> **made at CERN**, [the place where the web was born](https://home.cern/science/computing/birth-web)!
+
+![A sneak peek of Indico](https://raw.githubusercontent.com/pferreir/indico/new-readme/sneakpeek.gif)
+
+## What does it do?
+Indico's **main features** are:
+ * a powerful and flexible **hierarchical content management** system for events;
+ * a full-blown **conference organization workflow** with:
+   - 📢 **Call for Abstracts** and **abstract reviewing** modules;
+   - 📝 flexible **registration form** creation and configuration;
+   - 💰 integration with existing **e-payment systems**;
```

Comment:
I've never heard the term "e-payment" outside (old) indico feature descriptions. Maybe just "payment systems" is enough? It's pretty obvious that it's online :D

Reply:
Yeah, true.